examples/web-wagmi/src/profiles/UseMutualFollowers.tsx

```diff
@@ -49,14 +49,14 @@ export function UseMutualFollowers() {
 
       <article>
         <p>Select two profiles to see their mutual followers:</p>
-        <SelectProfileId onProfileSelected={(profile) => setSelectedProfileOne(profile)} />
-        <SelectProfileId onProfileSelected={(profile) => setSelectedProfileTwo(profile)} />
+        <SelectProfile onProfileSelected={(p) => setSelectedProfileOne(p)} />
```

Comment:
ah, I pretty much thought about the same thing while I was refactoring these tests to add error handling. Thanks!